values.yaml

```diff
@@ -284,6 +284,8 @@ redis:
 
 notary:
   enabled: true
+  tls:
```

Comment:
I think we should move the `tls` to the `ingress` section with a different name

Reply:
@ywk253100 I never knew you could put multiple TLS config! Great, i'll follow this approach.

Reply:
@ywk253100 This has been changed, ready for review